api_spec.yaml

```diff
@@ -912,7 +913,7 @@ paths:
                     $ref: '#/components/schemas/oauth2RedirectUri'
                 name:
                   type: string
-                  example: asdf
```

Comment:
if regenerateSecret is true, then we need to return the new secret in status OK response.